admin_guide/service_accounts.adoc

```diff
@@ -58,20 +57,89 @@ specified project.
 For example, to allow all service accounts in all projects to view resources in
 the *top-secret* project:
 
-====
 ----
 $ oc policy add-role-to-group view system:serviceaccounts -n top-secret
 ----
-====
 
 To allow all service accounts in the *managers* project to edit resources in the
 *top-secret* project:
 
-====
 ----
 $ oc policy add-role-to-group edit system:serviceaccounts:managers -n top-secret
 ----
-====
+// end::sa-user-names-and-groups[]
+
+
+[[admin-managing-service-accounts]]
+// tag::managing-service-accounts[]
+ifdef::openshift-online,openshift-origin,openshift-dedicated,openshift-enterprise[]
+== Managing Service Accounts
+
+Service accounts are API objects that exist within each project. To manage
+service accounts, you can use the `oc` command with the `sa` or `serviceaccount`
+object type or use the web console.
+
+To get a list of existing service accounts in the current project:
+
+----
+$ oc get sa
+NAME       SECRETS   AGE
+builder    2         2d
+default    2         2d
+deployer   2         2d
+----
+
+To create a new service account:
+
+----
+$ oc create sa robot
+serviceaccount "robot" created
+----
+endif::[]
+
+ifdef::atomic-registry[]
+[[managing-service-account-credentials]]
+== Managing Service Account Credentials
+endif::[]
+
+As soon as a service account is created, two secrets are automatically added to
+it:
+
+* an API token
+* credentials for the
+ifdef::openshift-enterprise,openshift-origin,openshift-online,openshift-dedicated[]
+OpenShift Container Registry
+endif::[]
+ifdef::atomic-registry[]
+internal registry
+endif::[]
+
+These can be seen by describing the service account:
+
+----
+$ oc describe sa robot
+Name:		robot
+Namespace:	project1
+Labels:		<none>
+Annotations:	<none>
+
+Image pull secrets:	robot-dockercfg-qzbhb
+
+Mountable secrets: 	robot-token-f4khf
+                   	robot-dockercfg-qzbhb
+
+Tokens:            	robot-token-f4khf
+                   	robot-token-z8h44
+
+----
+
+The system ensures that service accounts always have an API token and registry
+credentials.
+
+The generated API token and registry credentials do not expire, but they can be
+revoked by deleting the secret. When the secret is deleted, a new one is
+automatically generated to take its place.
```

Comment:
It's mentioned in https://docs.openshift.com/container-platform/3.5/dev_guide/secrets.html, though it could be better there.